doc/getting_started.mld

```diff
@@ -0,0 +1,78 @@
+{0 OCamlFormat Manual}
+
+{1 Getting started}
+
+{2 Installation}
+
+OCamlFormat can be installed with `opam`:
+
+{[
+opam install ocamlformat
+]}
+
+Alternatively, it can be built manually with:
+
+{[
+dune subst
+dune build @all
+]}
+
+{2 Formatting code!}
+
+To manually invoke OCamlformat the general command is:
+
+{[
+ocamlformat [OPTION]... [SRC]...
+]}
+
+The most common usecase involves using the Dune build system, once your project is correctly setup (see {{:https://dune.readthedocs.io/en/stable/formatting.html#formatting-a-project}Dune's manual}) you can reformat your project using:
```

Comment:
Would be cool to have some explanation on how it works. For instance:

- Does `fmt` only work with `ocamlformat`? (I think it works with `refmt` for Reason)
- Which file does it format? (only the ones in that have a dune rule I think)
- etc.

Reply:
done